docs/limitations.md

```diff
@@ -70,7 +70,7 @@
 - change outputs (indicated with paths, scripts in output section) must correspond to
   the active multisig wallet, and cannot be used to describe an unrelated (multisig) wallet.
 - derivation path for each cosigner must be known and consistent with PSBT
-- fixed: XFP values (fingerprints) for each of the co-signers must be unique (limitation removed)
+- XFP values (fingerprints) for each of the co-signers must be unique
```

Comment:
this belongs in some other commit, if at all

Reply:
removed - will open new PR with updated limitations